### ISSUE TITLE
Spam in journal of Failed to get application states: GDBus.Error:org.freedesktop.portal.Error.Failed

### ISSUE THREAD

**Issue by asegarra** [flatpak/xdg-desktop-portal-gtk#222]
Hi,

I am getting the following error in journal every minute if I have any flatpak installed on my system.

`xdg-desktop-por[5202]: Failed to get application states: GDBus.Error:org.freedesktop.portal.Error.Failed: Could not get window list: GDBus.Error:org.freedesktop.DBus.Error.AccessDenied: App introspection not allowed`

Fedora 30 Workstation
Gnome 3.32.2
xdg-desktop-portal : 1.4.2
xdg-desktop-portal-gtk : 1.4.0

**Comment by MysticalMount** [comment]
I can confirm this is also spamming my Fedora 30 journalctl every minute.

**Comment by UlfR** [comment]
Ubuntu 19.10beta, same spam

**Comment by matthiasclasen** [comment]
https://github.com/flatpak/xdg-desktop-portal/commit/a3fd03f852a27ac45fbaf740a965ce45af5f5390

**Comment by michaelachrisco** [comment]
I can confirm the same issue on 19.10.
```
/var/log/syslog

xdg-desktop-por[1370]: Failed to get application states: GDBus.Error:org.freedesktop.portal.Error.Failed: Could not get window list: GDBus.Error:org.freedesktop.DBus.Error.AccessDenied: App introspection not allowed
```

**Comment by seb128** [comment]
@matthiasclasen the commit fixes the spamming, but isn't the fact that it hits those errors in a GNIOME session a problem by itself?

**Comment by mmatoscom** [comment]
I've fixed as suggested in another issue,
$ gsettings set org.gnome.shell introspect true

**Comment by matthiasclasen** [comment]
> @matthiasclasen the commit fixes the spamming, but isn't the fact that it hits those errors in a GNIOME session a problem by itself?

The background monitoring won't work unless the portals get the window state information from the compositor. So yes, it is not ideal

**Comment by ziggyjosh16** [comment]
> I've fixed as suggested in another issue,
> $ gsettings set org.gnome.shell introspect true

Can someone please ELI5? I am also having this issue but would like to understand what is causing it to happen, or how to find out what is causing it.

**Comment by toddInPortland** [comment]
I have the same situation with spamming the syslog on Fedora 30. I use xfce plus some services and apps from kde and gnome. 

This solution from above failed as follows:
`$ gsettings set org.gnome.Shell introspect true
No such schema “org.gnome.Shell”`
This does work:
`rpm -e xdg-desktop-portal xdg-desktop-portal-gtk flatpak flatpak-session-helper flatpak-selinux`
I don't seem to be missing any services or options.

**Comment by ZanderBrown** [comment]
In danger of stating the obvious:
Your missing flatpak, and portals. Because you just deleted them.

Perhaps you didn't use any flatpak apps, fair enough, but increasingly things like web browsers make use for portals to provide functionality such as screen sharing — Sooner or later your probably going to miss that

**Comment by toddInPortland** [comment]
Yes, from what I've read, those are the packages that are ultimately responsible for the onece-per-minute reminder that

`Jun  2 04:46:48 archive2 journal[4884]: Failed to get application states: GDBus.Error:org.freedesktop.portal.Error.Failed: Could not get window list: Cannot invoke method; proxy is for the well-known name org.gnome.Shell without an owner, and proxy was constructed with the G_DBUS_PROXY_FLAGS_DO_NOT_AUTO_START flag
`

Most of my usage centers on gcc, g++, gdb, netbeans, slack, google, thunderbird, vnc, vlc, pycharm and an occasional  zoom. None of those seem to be lacking. I am sure the error message is my fault; probably in the way the system was configured.  But I could not find an explanation for the cause.

If you know of specific apps related the the collection above that rely  on flatpak, that would be interesting. As it is, I rely on dnf and rpm to ensure my packages have the proper dependencies installed. Currently, none of them require the packages above that I removed. Maybe if I was a full-stack developer...
